test.js

```diff
@@ -1,5 +1,12 @@
-var t = "hello";
+  var t = "hello";
+var b = null
 
-if (t === "hello') {
-    console.log('ok');
+if (t == "hello") {
+ console.log('ok');
 }
+
+if (true) {
```

Comment:
Reporter: CHECKSTYLE
Rule: eslint.rules.no-constant-condition
Severity: ERROR
File: /var/lib/openshift/57246f9589f5cf681e000139/app-root/data/jobs/QA-PR-CHECK/workspace/test.js L8
 

Unexpected constant condition. (no-constant-condition)

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>